spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -17,37 +17,144 @@
 // limitations under the License.
 // Contributors:
 // Steve Fontes
-[[architecture]]
 
 == Architecture
 
-This specification defines an easy way to instrument an application to take advantage of an opentracing.io Tracer implementation.
+This specification defines an easy way to allow an application running in a
```

Comment:
I haven't read it all.

Does the spec define a way to use span decorators?

Reply:
The spec defines default tags that will be added to created Spans. For other decorations, the programmer would have to do something like:
@Inject Tracer tracer;
....
tracer.activeSpan().setTag(...);